.github/workflows/PullRequestWorkflow.yaml

```diff
@@ -145,6 +145,59 @@ jobs:
             */build/reports/androidTests/**/${{ matrix.device.name }}
             */build/outputs/androidTest-results/**/${{ matrix.device.name }}
 
+  android-test-stable-cache:
```

Comment:
Adding a whole new workflow just for this feature is a bit overkill. Many of the tests being run don't depend on taking a picture or video, so there's not much value in running them again.

I would much prefer if this could just be set as an intent argument to enable/disable the cache-and-review workflow. Then we could just parameterize any of the tests that would use this workflow to run with cache-and-review and without.